awx/main/dispatch/worker/callback.py

```diff
@@ -90,29 +90,11 @@ def perform_work(self, body):
                 }
 
                 job_identifier = 'unknown job'
-                job_key = 'unknown'
                 for key, cls in event_map.items():
                     if key in body:
                         job_identifier = body[key]
-                        job_key = key
                         break
 
-                if settings.DEBUG:
-                    from pygments import highlight
-                    from pygments.lexers import PythonLexer
-                    from pygments.formatters import Terminal256Formatter
-                    from pprint import pformat
-                    if body.get('event') == 'EOF':
-                        event_thing = 'EOF event'
-                    else:
-                        event_thing = 'event {}'.format(body.get('counter', 'unknown'))
-                    logger.debug('Callback worker received {} for {} {}'.format(
-                        event_thing, job_key[:-len('_id')], job_identifier
-                    ))
-                    logger.debug('Body: {}'.format(
```

Comment:
This doesn't actually even print in the dev environment, because at some point we changed the default log level.  Also, pygments is *really* slow and is making the dev env seem much slower for large event throughput.